downstream/modules/eda/proc-eda-set-up-new-project.adoc

```diff
@@ -1,20 +1,20 @@
 [id="eda-set-up-new-project"]
 
-= Setting up a new project
+== Setting up a new project
```

Comment:
This should be = Setting up a new project and the extra = should be removed. See above for the explanation and don't hesitate to let me know if you need me to clarify further.